go/host/batchmanager/batch_manager.go

```diff
@@ -88,30 +87,25 @@ func (b *BatchManager) GetBatches(batchRequest *common.BatchRequest) ([]*common.
 	}
 
 	// We gather the batches by walking backwards from the final batch.
-	currentBatch := lastBatch
```

Comment:
A small optimisation, to avoid us grabbing the batch header _and_ the full batch for each batch.